src/etl/ReportingETL.cpp

```diff
@@ -933,38 +937,55 @@ ReportingETL::loadCacheAsync(uint32_t seq)
     {
         std::optional<ripple::uint256> start = cursors[i];
         std::optional<ripple::uint256> end = cursors[i + 1];
```

Comment:
is `cursors.back()` always going to be `{}`

Reply:
yes, we always push an empty optional to the front and back